wpeverest/everest-forms
# Feature - Form migrator for contact form 7

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces a new feature called **Form Migrator** which helps to import the form of **contact form 7** into our forms easily with one click.

### How to test the changes in this Pull Request:

1. Install and activate the contact form 7.
2. Create some forms with free fields.
3. Go to Everest form > tools menu > form migrator tab > Select contact form 7
4. verify the list of forms
5. Try to import single and multiple forms at once.

### Types of changes:


### Other information:


### Changelog entry

> Feature - Form migrator for contact form 7.
